docs/getting-started/openshift/index.md

```diff
@@ -0,0 +1,169 @@
+# Getting Started - OpenShift Routing
+
+This guide covers how Argo Rollouts integrates with [OpenShift Routing](https://docs.openshift.com/container-platform/4.7/networking/routes/route-configuration.html) for traffic management. This guide builds upon the [getting started guide](../../getting-started.md).
```

Comment:
better use latest version of the docs
https://docs.openshift.com/container-platform/4.11/networking/routes/route-configuration.html

Reply:
Thanks 👍